### PR TITLE
Avoid using InstallValue on non-plain objects

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

**Comment by fingolfin** [comment]
Is this package still maintained, @HereAround and @mohamed-barakat ?

If yes, it'd be great to have this merged and put in a release.

If not, would you like to withdraw it from distribution with GAP?

**Comment by HereAround** [comment]
Sadly, not much development here anymore @fingolfin.

All of the toric varieties functionality in this package is now available through OSCAR. Plus OSCAR comes with additional functionality that is not available here. In this regards, OSCAR is a proper superset to the gap ToricVarieties package.

(OSCAR is clearly behind regarding functionality of coherent sheaves, but that refers to the other GAP packages in this repo, which were never distributed with GAP.)

@mohamed-barakat Is there great interest in this package from your side? Or from users of the homalg_project? If so, then we can try to get this package working again. Otherwise, I am afraid it is time to let go and move on. 

Long story short: Unless @mohamed-barakat objects, I suggest to withdraw the ToricVarieties package from distribution with gap.

**Comment by mohamed-barakat** [comment]
This project was completely outsourced to @HereAround, so it is his decision. [ToricVarieties_project](https://github.com/homalg-project/ToricVarieties_project) is homalg-based and incompatible with our current philosophy. If we end up coming back to toric varieties we have to resurrect it as a CAP-based package, probably by a complete rewrite.

**Comment by HereAround** [comment]
@fingolfin Let us then withdraw the ToricVarieties package from GAP. What steps are needed on my end?

**Comment by HereAround** [comment]
No harm in merging this PR, and will thus do so. Thank you @fingolfin .
